purplet.config.ts

```diff
@@ -1,17 +1,10 @@
-import { ChatCommandHandler, defineConfig, OnEventHandler, TextCommandHandler } from 'purplet';
+import { defineConfig } from 'purplet';
 
 export default defineConfig({
   discord: {
     clientOptions: {
       intents: ['GUILD_MESSAGES'],
     },
-    commandGuilds: [],
+    commandGuilds: ['741460884307837041'],
```

Comment:
Shouldn't have this committed, instructions to add this are already in readme

Reply:
i think im gonna mess this up a lot, so i'm gonna add another env var for this (it was going to be added to the framework soon but not yet)